documentation/modules/ROOT/pages/connectors/postgresql.adoc

```diff
@@ -253,6 +253,13 @@ public interface Snapshotter {
                 .append(lineSeparator));
         return Optional.of(statements.toString());
     }
+
+    /**
```

Comment:
If you feel adventureous: WDYT about including this content from the actual Java source file. This is doable in AsciiDoc(tor): you put marker comments in the source file which denote a section of it, which then can be pulled in here. Avoids the redundancy.